.github/workflows/deploy-changed-users.yaml

```diff
@@ -0,0 +1,167 @@
+name: Update infrastructure users on AWS
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'infra/cluster_admins_*'
+      - 'infra/cluster_users'
+      - 'infra/biomage-read-only-group/**'
+      - 'infra/botkube-configmap.yaml'
+      - 'infra/cf-state-machine-role.yaml'
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'infra/cluster_admins_*'
+      - 'infra/cluster_users'
+      - 'infra/biomage-read-only-group/**'
+      - 'infra/botkube-configmap.yaml'
+      - 'infra/cf-state-machine-role.yaml'
```

Comment:
This is probably not what we want, this will apply the changes even when you submit a PR with the change.  Unless there are linting steps we can perform or a cold-run (as we do with e.g. cloudformation files) this is dangerous and should be removed.

Reply:
ok, makes sense